src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.Common/Altinn.App.Common.csproj

```diff
@@ -4,16 +4,16 @@
     <TargetFramework>net5.0</TargetFramework>
     <LangVersion>latest</LangVersion>
     <OutputType>Library</OutputType>
-    <AssemblyVersion>4.1.0</AssemblyVersion>
-    <FileVersion>4.1.0</FileVersion>
+    <AssemblyVersion>4.2.0</AssemblyVersion>
+    <FileVersion>4.2.0</FileVersion>
     <!-- SonarCloud requires a ProjectGuid to separate projects. -->
     <ProjectGuid>{7D2FF2B1-7F21-4934-9D77-DBCDC15BD8CC}</ProjectGuid>
   </PropertyGroup>
   
   <PropertyGroup>
     <!-- NuGet package properties -->
     <PackageId>Altinn.App.Common</PackageId>
-    <PackageVersion>4.1.0-alpha</PackageVersion>
+    <PackageVersion>4.2.0-alpha</PackageVersion>
```

Comment:
Why not 4.1.1-alpha?

Reply:
Upgrading the minor version following the SemVer standard. New feature -> updated minor. Bux fixes/patching -> updated patch

Reply:
Current version is 4.0 so I thought next minor version would be 4.1.*

Reply:
The slow calculate issue that is being tested has 4.1, so I bumped this to 4.2.. I guess only the alpha package is ready for that feature.